translate/translate.js

```diff
@@ -132,13 +153,17 @@ var program = module.exports = {
 
 cli
   .demand(1)
-  .command('detect <text>', 'Detect the language of the provided text', {}, function (options) {
-    program.detectLanguage(options.text, options.apiKey, utils.makeHandler(false));
+  .command('detect <input..>', 'Detect the language of the provided text', {}, function (options) {
+    program.detectLanguage(options.input, options.apiKey, utils.makeHandler(false));
   })
-  .command('list', 'List available translation languages.', {}, function (options) {
-    program.listLanguages(options.apiKey, utils.makeHandler());
+  .command('list [target]', 'List available translation languages.', {}, function (options) {
```

Comment:
Make clear what we mean by `target` here. (e.g. "To return language names in a language other than English, specify a target language.")